_sass/components/_accordions.scss

```diff
@@ -69,7 +69,7 @@
         content: '\0052';
       }
 
-      display: inline-block; 
+      display: inline-block;
```

Comment:
Expected item on line 72 to appear before line 68. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets